### ISSUE TITLE
Tests fail (with HTTP::Tiny 0.058?)

### ISSUE THREAD

**Issue by eserte** [kazeburo/Gazelle#26]
On my smoker systems the test suite of Gazelle-0.42 started to fail. Excerpt of the test log:
```
#   Failed test at /home/cpansand/.cpan/build/2016050421/Plack-1.0039-sjTgvY/blib/lib/Plack/Test/Suite.pm line 31.
#          got: 'Usage: $handle->write_request_header(method, request_uri, headers)
# '
#     expected: 'script_name='

#   Failed test at /home/cpansand/.cpan/build/2016050421/Plack-1.0039-sjTgvY/blib/lib/Plack/Test/Suite.pm line 43.
#          got: '599'
#     expected: '200'

#   Failed test at /home/cpansand/.cpan/build/2016050421/Plack-1.0039-sjTgvY/blib/lib/Plack/Test/Suite.pm line 44.
#          got: 'Internal Exception'
#     expected: 'OK'

#   Failed test at /home/cpansand/.cpan/build/2016050421/Plack-1.0039-sjTgvY/blib/lib/Plack/Test/Suite.pm line 46.
#          got: 'Usage: $handle->write_request_header(method, request_uri, headers)
# '
#     expected: 'Hello, name=miyagawa'
...
```

Statistical analysis suggests that the problems happens only with recent HTTP::Tiny versions (negative theta is bad) (@dagolden: FYI):
```
****************************************************************
Regression 'mod:HTTP::Tiny'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    36665540746119208.00
[1='eq_0.056']        0.0000          0.0000       0.00
[2='eq_0.058']       -1.0000          0.0000    -33165229470099532.00

R^2= 1.000, N= 62, K= 3
****************************************************************
```

**Comment by kazeburo** [comment]
Thank you for reporting.
Gazelle's test overwrite HTTP::Tiny::Handle::write_request_header In Gazelle/t/TestUtils.pm. That code caused this bug. I fix this asap.

**Comment by kazeburo** [comment]
I just released Gazelle-0.43 with bugfix. Thank you!
